demos/browser/app/meetingV2/video/VideoTile.ts

```diff
@@ -145,6 +145,10 @@ export class DemoVideoTile extends HTMLElement {
     console.log(`setting attendeeId to ${attendeeId}`);
     attendeeIdElement.innerHTML = attendeeId;
   }
+  public get attendeeId() {
+    const attendeeIdElement = this.querySelector('.video-tile-attendee-id');
+    return attendeeIdElement.innerHTML;
```

Comment:
nit: Isn't innerText better?. Also this seem like it can be a one liner :D

Reply:
Also can we just set a class variable to store the attendeeId for each video tile instead of having to use querySelector?